documentation/modules/changing-precopy-intervals.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="changing-precopy-intervals_{context}"]
+= Changing precopy intervals for warm migration
+
+The snapshots for the precopy stage of warm migration are created at 60-minute intervals by default.
+
+You can change the snapshot interval by patching the `forklift-controller` custom resource (CR).
```

Comment:
...ForkliftController custom resource (CR)
forklift-controller it is the name of the resource

Reply:
I fixed this alreadt.